sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/OrcTest.scala

```diff
@@ -15,18 +15,17 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.hive.orc
+package org.apache.spark.sql.execution.datasources
 
 import java.io.File
 
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe.TypeTag
 
 import org.apache.spark.sql._
-import org.apache.spark.sql.hive.test.TestHiveSingleton
 import org.apache.spark.sql.test.SQLTestUtils
 
-private[sql] trait OrcTest extends SQLTestUtils with TestHiveSingleton {
+abstract class OrcTest extends QueryTest with SQLTestUtils {
```

Comment:
When we have not moved the test suites of ORC, why we need to move out of Hive now?

Reply:
Unlike the previous ORC PR, #19651 intentionally uses the existing old Hive tests suites. This PR is a preparation for moving test cases.

Reply:
Since this PR is tiny, we can do it when the other PRs are merged.

Reply:
As you see, we can do this without merging other ORC PRs, and this will reduce the scope of review for the other PRs.